frame/lending/src/lib.rs

```diff
@@ -385,61 +448,150 @@ pub mod pallet {
 			amount: T::Balance,
 		},
 		/// Event emitted when user repays borrow of beneficiary in given market.
-		RepaidBorrow {
+		BorrowRepaid {
 			sender: T::AccountId,
 			market_id: MarketIndex,
-			beneficiary: T::AccountId,
+			beneficiary: T::AccountId, // REVIEW: Should beneficiary be removed?
 			amount: T::Balance,
 		},
 		/// Event emitted when a liquidation is initiated for a loan.
 		LiquidationInitiated {
 			market_id: MarketIndex,
 			borrowers: Vec<T::AccountId>,
 		},
-		/// Event emitted to warn that loan may go under collaterized soon.
-		SoonMayUnderCollaterized {
+		/// Event emitted to warn that loan may go under collateralized soon.
+		MayGoUnderCollateralizedSoon {
 			market_id: MarketIndex,
 			account: T::AccountId,
 		},
 	}
 
+	/////////////////////////////////////////////////////////////////////////
```

Comment:
//////////////
not sure i like these blocks. i think split should be done by modules, not comments. for the case. all complexity is moved from pallet.rs (you started doing it) and i did to auction.